<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Steganoimage</class>
 <widget class="QMainWindow" name="Steganoimage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steganoimage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>320</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image extraite 3 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_ajout_NB1_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une image à cacher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statut_image_originale">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image originale :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_ajout_NB1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une image à cacher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image extraite 2 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_extraction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Extraire les images</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image extraite 1 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_ajout_couleur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter image couleur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_originale">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>360</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_ajout_NB1_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une image à cacher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_sauvegarde">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder l'image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_occultation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Occulter les images</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_finale">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Première image à cacher :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deuxième image à cacher :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Troisème image à cacher :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_ouvrir">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvir une image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Image occultée :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bouton_ajout_couleur</sender>
   <signal>clicked()</signal>
   <receiver>label_image_originale</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
